webapp/sass/layout/_markdown.scss

```diff
@@ -3,6 +3,13 @@
 .markdown__heading {
     font-weight: 700;
     line-height: 1.5;
+
+    .emoticon {
```

Comment:
I'm sure I don't get it. Don't we need to have other emoticons follow the regular size?